validator_earning/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Flask
```

Comment:
lets pick specific versions maybe, so this doesn't break under us later

Reply:
Will fix requirements for all projects in a separate PR.